Cython/Plex/Traditional.py

```diff
@@ -147,12 +145,9 @@ def expect(self, c):
         if self.c == c:
             self.next()
         else:
-            self.error("Missing %s" % repr(c))
+            self.error("Missing {c}".format(c=repr(c)))
 
     def error(self, mess):
         """Raise exception to signal syntax error in regexp."""
-        raise RegexpSyntaxError("Syntax error in regexp %s at position %d: %s" % (
-            repr(self.s), self.i, mess))
-
-
-
+        raise RegexpSyntaxError("Syntax error in regexp %s at position %d: %s"
+                                % (repr(self.s), self.i, mess))
```

Comment:
Please don't change the code formatting here. It's much more readable and saves a lot of screen space to open a parenthesis on the right and have a single-step indent for the next line(s).

Reply:
Also: `%r`

Reply:
> It's much more readable and saves a lot of screen space to open a parenthesis on the right and have a single-step indent for the next line(s).

Out of curiosity, is this an editor-specific behavior?  It isn't clear to me how screen space is affected.

Regardless, not worth arguing about; reverting.

Reply:
The difference is between an indentation of 4 and an indentation of, say, 20, 40 or 60. What would you want to fit into a line that starts at character position 60? An indentation of 4 is entirely enough for the visual link, so I can see no reason to waste horizontal more space than that.